### ISSUE TITLE
Modal Close Button aria-label Not Correctly Translated for "de" Locale

### ISSUE THREAD

**Issue by masoudamri1959** [ant-design/ant-design#49554]
### Reproduction link

[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/vitejs-vite-pdnyb2?file=src%2FApp.tsx)

### Steps to reproduce

1. Set the locale to "de" (German) using the ConfigProvider component.
2. Create a modal using the Modal component.
3. Open the modal and inspect the close button using the Chrome Accessibility panel or with NVDA screen reader hovering over the close button.
4. Check the aria-label attribute for the close button in the modal.
5. The aria-label is currently in English.

### What is expected?

When the locale is set to "de" (German), the aria-label for the modal close button should be translated correctly to German. For instance, the translation for "Close" in German is "Schließen". Therefore, the aria-label should be "Schließen" instead of "Close".

Please fix this issue or add a feature that allows us to customize this text directly from the modal component.

![Modal UI](https://i.imgur.com/tp0Pj2q.png)

![Modal Chrome Html](https://i.imgur.com/7IcErq8.png)

![Modal Chrome Accessibility](https://i.imgur.com/doQoXIc.png)

In this images, the aria-label for the close button should be "Schließen" when the locale is set to "de".

### What is actually happening?

The aria-label for the modal close button is not translated when the locale is set to "de" (German).

| Environment | Info |
| --- | --- |
| antd | 5.17.4 |
| React | 18.3.1 |
| System | Windows 11 |
| Browser | Google Chrome 126.0.6478.61 |

**Comment by lijintao950830** [comment]
@afc163 这个问题是出在Antd-Modal的closable仅支持boolean类型，但在rc-dialog的closable支持object去修改aria-label，但不知道这样去做否合理？
![image](https://github.com/ant-design/ant-design/assets/60025467/99e0c846-61a2-4ee9-a704-1f0e719a2545)
[dialog](https://github.com/react-component/dialog/tree/master)

**Comment by EmilyyyLiu** [comment]
这个问题已经修改完成了，可以自行配置 aria-label
